deeplake/core/dataset/dataset.py

```diff
@@ -4764,7 +4763,6 @@ def max_view(self):
     def random_split(self, lengths: Sequence[Union[int, float]]):
         """Splits the dataset into non-overlapping :class:`~deeplake.core.dataset.Dataset` objects of given lengths.
         If a list of fractions that sum up to 1 is given, the lengths will be computed automatically as floor(frac * len(dataset)) for each fraction provided.
-        The split generated is only performant with enterprise dataloader which can be installed with ``pip install deeplake[enterprise]``.
```

Comment:
We should say: `This feature is only compatible with the high-performance dataloader.`

I asked @levongh to figure out what happens when this is used with OSS dataloader. I'm honestly not sure.